src/firebase/Firebase.tsx

```diff
@@ -28,14 +29,16 @@ const appCheck = initializeAppCheck(app, {
   isTokenAutoRefreshEnabled: true,
 });
 
-let appchecktoken: string;
-try {
-  async () => {
-    appchecktoken = (await getToken(appCheck, /* forceRefresh= */ false)).token;
-  };
-} catch (err) {
-  alert(err);
+export async function getAppcheck() {
+  let appchecktoken: string;
+  let appchecktokenresult: AppCheckTokenResult;
+  try {
+    appchecktokenresult = await getToken(appCheck, /* forceRefresh= */ false);
+    appchecktoken = appchecktokenresult.token;
+    if (appchecktoken !== "") return appchecktoken;
```

Comment:
```suggestion
  try {
    res = await getToken(appCheck, /* forceRefresh= */ false);
    const appCheckToken = res.token;
    if (appCheckToken !== "") return appCheckToken;
```
Cleaned up. Zou nog wel ff moeten testen als je dit mooi vind om te committen